flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/DeclarativeSlotManagerTest.java

```diff
@@ -1347,7 +1351,7 @@ public void testAllocationUpdatesIgnoredIfSlotMarkedAsPendingForOtherJob() throw
                         .setSlotTracker(slotTracker)
                         .buildAndStart(
                                 ResourceManagerId.generate(),
-                                ComponentMainThreadExecutorServiceAdapter.forMainThread(),
+                                Executors.directExecutor(),
```

Comment:
Could you help me to understand why we need this change?

Reply:
It's an unused change now, reverted